src/modules/Dashboard/features/Applet/Schedule/ImportSchedulePopup/ImportSchedulePopup.tsx

```diff
@@ -0,0 +1,120 @@
+import { useEffect, useState } from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+
+import { FileUploader, Modal, SubmitBtnColor } from 'shared/components';
+import { ImportedFile } from 'shared/components/FileUploader/FileUploader.types';
+import { StyledBodyLarge, StyledModalWrapper } from 'shared/styles';
+
+import { getScreens, invalidFileFormatError, uploadLabel } from './ImportSchedule.const';
+import { ImportSchedulePopupProps, Steps } from './ImportSchedulePopup.types';
+
+export const ImportSchedulePopup = ({
+  isIndividual = false,
+  open,
+  onClose,
+}: ImportSchedulePopupProps) => {
+  const { t } = useTranslation('app');
+
+  const appletName = 'Pediatric Screener';
```

Comment:
Pls use real data appletName
in Schedule.tsx
 const { result: appletData } = applets.useAppletData() ?? {};
<Legend legendEvents={preparedEvents} appletName={appletData?.displayName || ''} />
in Legend.tsx
export const Legend = ({ legendEvents, appletName }: LegendProps) => { ...
and pass appletName as a property